samples/nrf9160/fmfu_smp_svr/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
```

Comment:
```suggestion
# Copyright (c) 2021 Nordic Semiconductor
```

Reply:
Seems a rouge commit got rebased into it :/